src/hooks/wallets/mpc/useMPC.ts

```diff
@@ -18,9 +18,10 @@ export const useInitMPC = () => {
   useInitSocialWallet()
 
   useEffect(() => {
-    if (!chain || !onboard) {
+    if (!chain || !onboard || !isSocialWalletOptions(SOCIAL_WALLET_OPTIONS)) {
```

Comment:
Can we do this check at the entry point to this flow instead?
I.e. not even show the Google button if these env vars aren't set. This is important for forks.

Reply:
I think the config service toggle only changes if the button is enabled or not. We could use the env vars to decide if the feature should be enabled or not though.